lib/builders/testArgBuilder.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (value) => {
+    let args = [];
+    if(!value){
+        return args;
+    }
+    if (value.settings) {
+        args = args.concat(['--settings', value.settings]);
+    }
+    if (value.listTests) {
+        args = args.concat(['--list-tests']);
+    }
+    if (value.testAdapterPath) {
+        args = args.concat(['--test-adapter-path', value.testAdapterPath]);
+    }
+    if (value.logger) {
+        args = args.concat(['--logger', value.logger]);
+    }
+    if (value.configuration) {
+        args = args.concat(['--configuration', value.configuration]);
+    }
+    if (value.framework) {
+        args = args.concat(['--framework', value.framework]);
+    }
+    if (value.output) {
+        args = args.concat(['--output', value.output]);
+    }
+    if (value.diag) {
```

Comment:
Diagnostic Logging does jump out at me as what this does. I guess where it looks like a pass through its OK, but a more descriptive name might be desirable.